CircleCI is failing unit tests in Localization

https://circleci.com/gh/scottleedavis/mattermost-plugin-remind/tree/master
https://circleci.com/gh/scottleedavis/mattermost-plugin-remind/220

```
--- FAIL: TestTranslationsPreInit (0.00s)
    --- PASS: TestTranslationsPreInit/failure_to_get_bundle_path (0.00s)
        i18n_test.go:36: PASS:	GetBundlePath(string)
    --- FAIL: TestTranslationsPreInit/failure_to_read_i18n_directory (0.00s)
        require.go:172: 
            	Error Trace:	i18n_test.go:52
            	Error:      	Error message not equal:
            	            	expected: "unable to read i18n directory: readdirent: invalid argument"
            	            	actual  : "unable to read i18n directory: readdirent: not a directory"
            	Test:       	TestTranslationsPreInit/failure_to_read_i18n_directory
        panic.go:406: PASS:	GetBundlePath(string)
    --- PASS: TestTranslationsPreInit/no_i18n_files (0.00s)
        i18n_test.go:69: PASS:	GetBundlePath(string)
    --- PASS: TestTranslationsPreInit/various_i18n_files (0.00s)
        mock.go:392: Loaded translation file filename en.json
        mock.go:392: Loaded translation file filename es.json
        mock.go:392: Failed to load translation file filename invalid.json err no language found in "invalid.json"
        i18n_test.go:109: PASS:	GetBundlePath(string)
        i18n_test.go:109: PASS:	LogError(string,string,string,string,string)
        i18n_test.go:109: PASS:	LogDebug(string,string,string)
```

--- a/server/i18n_test.go
+++ b/server/i18n_test.go
@@ -49,7 +49,8 @@ func TestTranslationsPreInit(t *testing.T) {
 		p := &Plugin{}
 		p.API = api
 		err = p.TranslationsPreInit()
-		require.EqualError(t, err, fmt.Sprintf("unable to read i18n directory: readdirent: invalid argument"))
+		require.True(t, err.Error() == "unable to read i18n directory: readdirent: invalid argument" ||
+			err.Error() == "unable to read i18n directory: readdirent: not a directory")
 	})
 
 	t.Run("no i18n files", func(t *testing.T) {

